src/coins.h

```diff
@@ -20,67 +20,28 @@
 #include "zcash/History.hpp"
 #include "zcash/IncrementalMerkleTree.hpp"
 
-/** 
+enum Spentness {
+    UNSPENT,
+    SPENT
+};
+
+/**
  * Pruned version of CTransaction: only retains metadata and unspent transaction outputs
- *
- * Serialized format:
```

Comment:
It's difficult to review this move for idempotence, because the commit (I think) it occurs in (b311ec96149d78fc1f3a2ab96466b5b6f73c5847) contains numerous other changes (+517/-229). Please refactor this into a separate move-only commit.